site/content/docs/5.3/getting-started/webpack.md

```diff
@@ -148,7 +153,10 @@ Importing Bootstrap into Webpack requires the loaders we installed in the first
 1. **Set up the loaders in `webpack.config.js`.** Your configuration file is now complete and should match the snippet below. The only new part here is the `module` section.
 
    ```js
+   'use strict'
+
    const path = require('path')
+   const HtmlWebpackPlugin = require('html-webpack-plugin')
```

Comment:
we need autoprefixer here too and maybe in the install instruction if it's not mentioned

Reply:
Good catch you're right. Let me fix it

Reply:
And probably postcss and cli too.

Reply:
I know they are not in it but won't the file error out if the devDependencies are not installed?

Haven't read through these guides, just thinking out loud here.